deployment/CP03ISSM_Deploy.csv

```diff
@@ -1,50 +1,50 @@
+KN-224,,AT-31,,CP03ISSM-RIC21-00-CPMENG000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGCON-NCPM02-00005,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID26-00-DCLENG000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGCON-NDCL26-00011,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID26-04-VELPTA000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-VELPTA-11397,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID26-06-PHSEND000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-PHSEND-P0087,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID26-07-NUTNRB000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-NUTNRB-00268,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID26-08-SPKIRB000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-SPKIRB-00270,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID27-00-DCLENG000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGCON-NDCL27-00005,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID27-01-OPTAAD000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-OPTAAD-00187,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID27-02-FLORTD000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-FLORTD-01106,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID27-03-CTDBPC000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-CTDBPC-50003,40.3619,-70.878,,7,94,
+KN-224,,AT-31,,CP03ISSM-RID27-04-DOSTAD000,1,1,2014-12-14T18:53:00,2015-10-30T19:30:00,CGMCP-03ISSM-00001,,CGINS-DOSTAD-00389,40.3619,-70.878,,7,94,
+KN-224,,WJ20141230,,CP03ISSM-SBC11-00-CPMENG000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGCON-BCPM01-00008,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD11-00-DCLENG000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGCON-BDCL11-00008,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD11-01-MOPAK0000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGINS-MOPAK0-99999,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD11-02-HYDGN0000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGINS-HYDGN0-99999,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD11-06-METBKA000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGINS-METLGR-00013,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD12-00-DCLENG000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGCON-BDCL12-00008,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD12-03-HYDGN0000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGINS-HYDGN0-99999,40.3619,-70.878,,0,94,
+KN-224,,WJ20141230,,CP03ISSM-SBD12-04-PCO2AA000,1,1,2014-12-14T18:53:00,2014-12-30T22:30:00,CGMCP-03ISSM-00001,,CGINS-PCO2AA-33183,40.3619,-70.878,,0,94,
```

Comment:
Per discussions, the Buoy and NSIF should use the "event" date/time (2014-12-29T16:30:00) for stopDateTime.
All else looks good.

Reply:
Updated date/time for buoy & NSIF.